Bottom: 8828d525ae505976d6eade1fa780009dbd0b9e30
Top:    fe9ffab357b42efea7a80c06160dfe3ff8440642
Author: Jon Smirl <jonsmirl@smirl2.(none)>
Date:   2013-06-20 10:12:51 -0400

stash work


---

diff --git a/feeds.conf.default b/feeds.conf.default
index 67c8ddb..6496089 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,4 +1,4 @@
-src-svn packages svn://svn.openwrt.org/openwrt/packages
+src-git packages git clone git://git.openwrt.org/packages.git
 src-svn xwrt http://x-wrt.googlecode.com/svn/trunk/package
 src-git luci git://nbd.name/luci.git
 src-git routing git://github.com/openwrt-routing/packages.git
diff --git a/package/mac80211/Makefile b/package/mac80211/Makefile
index 4bd6488..f432787 100644
--- a/package/mac80211/Makefile
+++ b/package/mac80211/Makefile
@@ -234,7 +234,6 @@ $(call KernelPackage/rt2x00/Default)
   DEPENDS+= @(PCI_SUPPORT||TARGET_ramips) +kmod-rt2x00-lib +kmod-eeprom-93cx6
   TITLE+= (PCI)
   FILES:= \
-	$(PKG_BUILD_DIR)/drivers/net/wireless/rt2x00/rt2x00pci.ko \
 	$(PKG_BUILD_DIR)/drivers/net/wireless/rt2x00/rt2x00mmio.ko
   AUTOLOAD:=$(call AutoLoad,25,rt2x00mmio rt2x00pci)
 endef
